setup.py

```diff
@@ -30,6 +30,7 @@
     'python-dateutil>2,<3',
     'six>1,<2',
     'setuptools',
+    'sshuserclient'
```

Comment:
Could we pin this please to `sshuserclient>0.3,<1`? Aside from setuptools would like to keep dependencies bound to major versions just-in-case.